packages/lndr/recent-transaction/index.ts

```diff
@@ -1,13 +1,13 @@
 //Using transactions?values={x}
-export default class TransactionHistory {
+export default class RecentTransaction {
   debtor: string
   creditor: string
   amount: number
   ucac: string
   doesUserOweFriend: boolean
```

Comment:
This is not a property of the transaction - instead make a `RecentTransaction.direction(address)` method that takes the user's address and returns `'lend'` or `'borrow'`